CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -202,6 +202,10 @@ extension GiniScreenAPICoordinator {
         switch screenAPINavigationController.topViewController {
         case is CameraViewController:
             trackingDelegate?.onCameraScreenEvent(event: Event(type: .exit))
+            //TODO: rethink this when you add Camera access screen events!
```

Comment:
[Clarifcation]: would you like to keep this inline comment?

Reply:
I would like to keep it until I add those events for the camera access screen. It will be much easier for me.